src/patient/patient.resources.ts

```diff
@@ -0,0 +1,19 @@
+import { type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+
+export const usePatientAge = (patienUuid: string) => {
+  const customRep = 'custom:(age)';
+  const url = `${restBaseUrl}/person/${patienUuid}?v=${customRep}`;
+  const { data, error, isLoading, mutate } = useSWR<FetchResponse<{ age: number }>>(url, openmrsFetch);
+  return {
+    age: data?.data?.age,
+    error,
+    isLoading,
+    mutate,
+  };
+};
+
+export function concatAgePatientDisplay(input: string, age: number): string | null {
+  const attrIndex = input.lastIndexOf(')');
+  return input.slice(0, attrIndex) + ', Age: ' + age + input.slice(attrIndex);
```

Comment:
Minor, but please use "t()" here to localize "Age"

Reply:
Also, as this is a  function around formatting and display, let's put this in the patient-info-cell component, not the resource.

Reply:
Sorry, one other thing... but if the lastIndexOf(')') returns -1 for some reason, let's not do the slice and simply append "Age: " + age to the end?  (Just in case the display format changes at some point.

Reply:
Thanks @mogoodrich , incorporated your suggestions